audit/audit_contract.go

```diff
@@ -119,12 +121,14 @@ func ListByCID(epoch int, cid []byte) [][]byte {
 	var buf interface{} = epoch
 
 	prefix := append(buf.([]byte), cid...)
+	ctx := storage.GetReadOnlyContext()
```

Comment:
Let's move it up to the top of the function as it everywhere else.

Reply:
moved